src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: Avenir;
+  font-style: book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-weight: 300;
+}
+
+form[data-qa="big"] {
```

Comment:
Add some class to a form and style it using class selector
![image](https://user-images.githubusercontent.com/96409563/217478546-01458eb8-5788-424d-bd15-7240193c88e2.png)